transform/kubernetes/kubernetes.go

```diff
@@ -435,20 +461,70 @@ func (k *KubernetesTransformPlugin) getKubernetesTransforms(obj unstructured.Uns
 		}
 		replicaSet := &appsv1.ReplicaSet{}
 		err = json.Unmarshal(js, replicaSet)
+		if err != nil {
+			return nil, err
+		}
 
 		patches, err := renamePVCs(replicaSet.Spec.Template.Spec.Volumes, k.PVCRenameMap, pvcPathGenericString)
 		if err != nil {
 			return nil, err
 		}
 		jsonPatch = append(jsonPatch, patches...)
 	}
+	if ocpRoleBindingGK == obj.GetObjectKind().GroupVersionKind().GroupKind() {
+		js, err := obj.MarshalJSON()
+		if err != nil {
+			return nil, err
+		}
+		rb := &ocprbacv1.RoleBinding{}
+
+		err = json.Unmarshal(js, rb)
+		if err != nil {
+			return nil, err
+		}
+		for i, subj := range rb.Subjects {
+			if subj.Kind == "ServiceAccount" && subj.Namespace == rb.Namespace {
+				subjPath := fmt.Sprintf(roleBindingSubject, i)
+				patch, err := jsonpatch.DecodePatch([]byte(fmt.Sprintf(opRemove, subjPath)))
+				if err != nil {
+					return nil, err
+				}
+				jsonPatch = append(jsonPatch, patch...)
+			}
+		}
+	}
+	if roleBindingGK == obj.GetObjectKind().GroupVersionKind().GroupKind() {
+		js, err := obj.MarshalJSON()
+		if err != nil {
+			return nil, err
+		}
+		rb := &rbacv1.RoleBinding{}
+
+		err = json.Unmarshal(js, rb)
+		if err != nil {
+			return nil, err
+		}
+		for i, subj := range rb.Subjects {
```

Comment:
@jmontleon this looks super repetitive. Maybe you could put this logic in a function, pass the subjects, loop through them, and pass back the patches.

Reply:
@jmontleon I'd love to but Subjects in k8s rolebindings are []k8s.io/api/rbac/v1.Subject. On openshift rolebindings are [][]k8s.io/api/core/v1.ObjectReference